airgun/settings.py

```diff
@@ -60,6 +61,8 @@ def __init__(self):
         self.screenResolution = None
         self.tunnelIdentifier = None
         self.tags = None
+        self.browserName = None
+        self.javascriptEnabled = None
```

Comment:
browserName is needed because tests in selenium grid are running only on chrome browser but in the selenium grid is also firefox browser so it is to avoid start test on some firefox browser and say this test will run in chrome. The tests in firefox are failing because of security reasons.
I will delete javascriptEnabled.